doc/source/man.rst

```diff
@@ -327,52 +325,52 @@ A scan can be run by specifying the profile to use and where to write the CSV fi
 
 ``--scan-dirs dir1 dir2``
 
-  The list of directories on remote systems to scan for products. This option is intended to help scope a scan for systems with very large file system under the root directory. You may provide a list of directories or a file where each item is on a separate line.
+  Contains the list of directories on remote systems to scan for products. This option is intended to help scope a scan for systems with a very large file system under the root directory. You can provide multiple values for this option, with each value separated by a space, or provide a path to a file that contains a list of directories, where each directory is on a separate line.
 
 ``--cache``
 
-  This argument can be used if a profile has previously been used for a discovery and nothing new needs to be found during the scan
+  Restricts the scope of the scan to the hosts that were discovered in the previous scan. Use this option to discover software on hosts that were discovered in a previous scan. Do not use this option to scan for new hosts.
 
 ``--vault=vault_file``
 
-  This contains the path of the file that contains the vault password. If this option is used the file should be limited in access on the system.
+  Contains the path of the file that contains the vault password. Because the encrypted Rho data could contain sensitive information, make sure that this vault password file is stored in a location that has limited access.
 
 ``--logfile=log_file``
 
-  This contains the path of the file for writing the scan log.
+  Contains the path of the log file for this instance of the ``rho scan`` command.
 
 ``--ansible-forks=num_forks``
 
-  This value is used to determine the number of systems to scan in parallel. The current default is 50 concurrent connections.
+  Sets the number of systems to scan in parallel. The default number is 50 concurrent connections.
 
 Options for All Commands
 ------------------------
 
-A the following option is allowed with every command for rho.
+The following options are available for every Rho command.
```

Comment:
`s/Rho/rho/` again -- if we are sticking with `rho`

Reply:
See your first comment about Rho/rho, existential crisis, etc.